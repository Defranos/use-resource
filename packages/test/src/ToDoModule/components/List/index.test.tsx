import React from "react";
import renderer from "react-test-renderer";
import { ITodo } from "ToDoModule";
import List from ".";

const mockList: ITodo[] = [
  {
    id: 1,
    name: "My first todo task",
    isComplete: false,
  },
];

describe("[TODO] => List", () => {
  it("Should render properly", () => {
    const tree = renderer.create(<List todoList={mockList} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
