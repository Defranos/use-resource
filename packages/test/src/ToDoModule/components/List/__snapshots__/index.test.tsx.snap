// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[TODO] => List Should render properly 1`] = `
<div>
  <div
    style={
      Object {
        "margin": "1em",
      }
    }
  >
    <div
      className="makeStyles-wrapper-1 makeStyles-wrapper-3"
    >
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "fontSize": "1.2em",
          }
        }
      >
        <div
          style={
            Object {
              "flex": 1,
            }
          }
        >
          My first todo task
        </div>
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root makeStyles-icon-2"
          focusable="false"
          onClick={[Function]}
          viewBox="0 0 24 24"
        >
          <path
            d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6v12zM19 4h-3.5l-1-1h-5l-1 1H5v2h14V4z"
          />
        </svg>
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root makeStyles-icon-2"
          focusable="false"
          onClick={[Function]}
          viewBox="0 0 24 24"
        >
          <path
            d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
